spec/symbols_spec.rb

```diff
@@ -43,4 +51,31 @@ def expect_transformation(klass, *input, arg_signature)
 
     expect_transformation(SampleJob, *input, arg_signature)
   end
+
+  it "works with subclasses that don't include the module directly" do
+    input = [1, "x" => { "y" => 2, z: { "foo bar" => 0 } }]
+    arg_signature = [[1], { x: { y: 2, z: { :"foo bar" => 0 } } }]
+
+    expect_transformation(SubclassWithoutIncludeJob, *input, arg_signature)
+  end
+
+  if RUBY_VERSION >= "2.1.0"
```

Comment:
The comment or the actual test? The test is checking that on 2.1+, required keyword arguments behave as expected. It's behind a `RUBY_VERSION` check because `def foo(x:)` is a syntax error pre-2.1.